org.testeditor.web.backend.persistence/src/main/java/org/testeditor/web/backend/persistence/workspace/WorkspaceResource.xtend

```diff
@@ -105,6 +102,37 @@ class WorkspaceResource {
 		return pullResponse
 	}
 
+	private def void completePullResponseForDiff(PullResponse pullResponse, DiffEntry diff,
+		OpenResources openResources) {
+		val oldDiffPath = diff.getPath(Side.OLD)
+		val newDiffPath = diff.getPath(Side.NEW)
+		pullResponse.changedResources.addIf(oldDiffPath, [
+			oldDiffPath.isRelevantUnreportedChangedResource(openResources, pullResponse)
```

Comment:
The method name threw me off a bit, especially the "unreported", until I realized that this was referring to pullResponse potentially already containing the given path. Don't have an improvement suggestion though, maybe I'm just a bit slow this morning :)

Unrelated note: I don't think there's actually any benefit in using a lambda here, is there? I think `addIf` could just as well accept a boolean, plain and simple.